sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_options.hpp

```diff
@@ -51,7 +51,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
      * @brief Use to send request to the 7.2 version of Key Vault service.
      *
      */
-    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const ServiceVersion V7_2;
+    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const ServiceVersion V7_3;
```

Comment:
We don't want to just update the version, we need to *add* a new enum value. Users should be able to use the latest SDK to talk to older supported service versions since v1 shipped, since there should be no breaking changes.

And then specify that in client options.